wiki/Tournaments/OWC/2015/de.md

```diff
@@ -1,215 +1,480 @@
 11. Wenn weniger als 4 Spieler eines Teams anwesend sind, kann die Partie um maximal 10 Minuten verschoben werden.
 12. Das Austauschen von Spielern während der Partie ist erlaubt.
 13. Lag ist kein gültiger Grund, um das Ergebnis einer Beatmap zu nichtigen.
-14. Alle Spieler sollen das Match flüssig und ohne Delays abhalten. Exzessive Matchverzögerungen, welche vom jeweiligen Spieler verursacht werden, können bestraft werden.
-15. Wenn ein Spieler außerhalb der Matches die Verbindung verlieren sollte und das Team keine Alternative zur Verfügung hat, kann das Match bis zu maximal 10 Minuten verschoben werden.
-16. Alle Spieler und Schiedrichter müssen mit Respekt behandelt werden. Anweisungen von den Schiedsrichtern und Turnier Management ist Folge zu leisten. Endgültige Entscheidungen können nicht beanstandet werden.
-17. Unterbrechung des Spiels durch unfaires Spielen, Auswählen von unangemessenen WarmUp Beatmaps (siehe unten), Beleidigen oder Provozieren von anderen Spielern oder Schiedsrichtern, Verzögern der Matches oder absichtiliches unangemessenes Verhalten wird strengstens bestraft.
-18. Die Multiplayer Chaträume unterliegen den [osu! Community Rules](/wiki/Rules). Alle Chat Regeln gelten daher auch für die Multiplayer Chaträume.
-    1. Das Verletzen der Chatregeln führt zu Silences. Gesilencte Spieler können nicht an Multiplayerpartien teilnehmen und müssen für die Zwischenzeit ausgewechselt werden.
-19. In der Gruppenphase wird das 'Win by default' (zu deutsch 'kampflos gewonnen') als ein Sieg bei 4:0 und +2.5 Score Difference Ratio.
-20. Unerwartete Vorkommnisse werden vom Turnier Management behandelt. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
+14. Alle Spieler sollen die Partie flüssig und ohne Verzögerungen abhalten. Exzessive Verzögerungen der Partie, welche vom jeweiligen Spieler verursacht werden, können bestraft werden.
+15. Wenn ein Spieler außerhalb der Partien die Verbindung verlieren sollte und das Team keine Alternative zur Verfügung hat, kann die Partie bis zu maximal 10 Minuten verschoben werden.
+16. Alle Spieler und Schiedrichter müssen mit Respekt behandelt werden. Anweisungen von den Schiedsrichtern und Turnier-Management ist Folge zu leisten. Endgültige Entscheidungen können nicht beanstandet werden.
+17. Unterbrechung des Spiels durch unfaires Spielen, Auswählen von unangemessenen Aufwärm-Beatmaps (siehe unten), Beleidigen oder Provozieren von anderen Spielern oder Schiedsrichtern, Verzögern der Partien oder absichtiliches unangemessenes Verhalten wird strengstens bestraft.
+18. Die Multiplayer Chaträume unterliegen den [osu! Community-Regeln](/wiki/Rules). Alle Chat Regeln gelten daher auch für die Multiplayer Chaträume.
+    1. Das Verletzen der Chatregeln führt zu Stummschaltungen. Stummgeschaltene Spieler können nicht an Multiplayerpartien teilnehmen und müssen für die Zwischenzeit ausgewechselt werden.
```

Comment:
```suggestion
    1. Das Verletzen der Chat-Regeln führt zu Stummschaltungen. Stummgeschaltete Spieler können nicht an Multiplayerpartien teilnehmen und müssen für die Zwischenzeit ausgewechselt werden.
```
FYI: The first suggestion for `chat rules` is Chat-Regeln. Chatregeln comes as the second one.